### ISSUE TITLE
Error in config.sample for feature influxdb2?

### ISSUE THREAD

**Issue by rucksman** [datenschuft/SMA-EM#67]
I am running smaemd since quite some time and want to log now data into influxdb. I started by checking the section for influxdb2 in the config.sample, but I am confused:

```
#pvdata
pvmeasurement=SMAWR
pvvields=AC Power,AC Voltage,grid frequency,DC Power,DC input voltage,daily yield,total yield
#How frequently to send updates over (defaults to 20 sec)
min_update=30

#pvdata
pvmeasurement=SMAWR
pvfields=AC Power,grid frequency,DC input voltage,daily yield,total yield,Power L1,Power L2,Power L3

```
Looks like both sections are double, but on the other hand entries in "pvfields" are different. Can you please explain these two sections?

**Comment by sellth** [comment]
Looks like a paragraph was unintentionally doubled here. In any case the value should be called "pvfields" and defines which values queried by pvdata should be saved in your InfluxDB.

Fixed with #68

**Comment by datenschuft** [comment]
pr merged
